compile-jade.sh

```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-
-# I find it better to watch and compile each file individually,
-# rather than watching an entire directory. Edits to the data source,
-# i.e. json file backing a jade file will not trigger the recompile.
-# In that case you will want to cancel, and re-run this script to force
-# a recompile.
```

Comment:
Also, to address your comment here in the future we can incorporate this package: https://www.npmjs.com/package/gulp-jade-find-affected